Packs/WALLIXBastion/Integrations/WAB/WAB.py

```diff
@@ -520,7 +543,12 @@ def notify_approvers_linked_to_approval_request(self, args: Dict[str, Any]):
         body = assign_params(id=approval_request_notify_post_id)
         response = self._http_request("post", "/approvals/requests/notify", json_data=body)
 
-        return CommandResults(readable_output="Success!", raw_response=response)
+        return CommandResults(
+            outputs_prefix="WAB.approval_request_notify_post_response",
+            outputs_key_field="",
+            outputs=response,
+            raw_response=response,
+        )
```

Comment:
Thank you for this observation. In that case, the response is a json object which does not contain an id field (it returns a count of emails. what I could do is augment the response object with the id passed in argument to the request.

readable_output was there before as a default message because the response did not contain a body. Now that it does, should I leave it anyway? If so, that would be the case for all other commands which return data in response.

Reply:
Following my suggestion on the outputs_key_field: do we agree that if the outputs is a list of items, it does not make sense to set an outputs_key_field which has the same value for all items, as they would override each other? If that is correct, only responses which return a single objects can use the parameter id as outputs key.

Reply:
I ended up implementing this option and adding outputs_key_field where possible, i.e. when the response is not an array. In this case, I guess the only option would be to wrap the array in an object containing the key field, like 
```json
{"key_field": "value", "items", [...]}
```
but that would break the interface of many existing commands.

Reply:
Can you share an example of the response that comes back from calling the API of the command?

And regarding the readable_outputs, you can certainly use `tableToMarkdown`, this is a function that analyzes both a single json object and a list of json

If you think it is not important to display the all response in warroom as markdown using the `tableToMarkdown` function, then it is better to leave the message that was until now as success so that the client can see the result of the command in warroom

Reply:
Noted for tableToMarkdown. Will push an update.

The response from the quoted command looks like this:

```json
{
  "emails_count": 4
}
```

With the last update I made, it will now look like this, including the id passed in argument:

```json
{
  "approval_request_notify_post_id": "xxxx",
  "emails_count": 4
}
```

My last comment about lists without a key was referring to other commands such as wab-get-certificates-on-device which returns a list of items without a unique id.